apps/blockchain/scripts/sync_with_infura.ex

```diff
@@ -136,10 +142,10 @@ current_block =
 parents = SyncWithInfura.get_parents(current_block, db, [])
 
 tree =
-  Enum.reduce(parents, empty_tree, fn block, tree ->
-    {:ok, tree} = Blockchain.Blocktree.verify_and_add_block(tree, chain, block, db)
-    Logger.info("Verified Saved Block #{block.header.number}")
-    tree
-  end)
+  case MerklePatriciaTree.DB.get(db, "current_block_tree") do
```

Comment:
What if there is a value of "current_block_tree" in the DB but you want to start from the first block?

Reply:
I've just been removing the `db` directory. Ideally we replace this script with a CLI tool that's pulling blocks from ex_wire soon.